_rules/auto-update-text-efbfc7.md

```diff
@@ -0,0 +1,318 @@
+	<input type="button" onclick="changeFrequency(document.getElementById('interval').value)" value="Change frequency" />
+
+	<script type="text/javascript" src="/test-assets/efbfc7/script.js"></script>
+</body>
+```
+
+#### Passed Example 5
+
+This `span` element contains text content that automatically updates multiple times without user intervention and the location of an [instrument][] to control the updates is clearly available.
+
+```html
+<body onload="startUpdates()">
+	<p>
+		The W3C Web Accessibility Initiative (WAI) develops standards and support materials to help you understand and
+		implement accessibility.
+	</p>
+
+	<p>Random number: <span id="target">1</span></p>
+	<p>To control the random number updates activate the "Control updates" button.</p>
+	<input type="button" onclick="openModal()" value="Control updates" />
+
+	<div
+		style="
+      display: none;
+      position: fixed;
+      top: 2em;
+      left: 10em;
+      background-color: #505050;
+      color: white;
+      padding: 1em;
+      padding-top: 0em;
+    "
+		id="overlay"
+	>
+		<p>Control updates</p>
+		<input type="button" id="control" onclick="toggleUpdates()" value="Pause updates" />
+		<input type="button" onclick="hide()" value="Hide updates" />
+		<button onclick="closeModal();">Dismiss</button>
+	</div>
+
+	<script type="text/javascript">
+		function openModal() {
+			document.getElementById('overlay').style.display = 'block'
+		}
+		function closeModal() {
+			document.getElementById('overlay').style.display = 'none'
+		}
+	</script>
+	<script type="text/javascript" src="/test-assets/efbfc7/script.js"></script>
+</body>
+```
+
+### Failed
+
+#### Failed Example 1
+
+This `span` element contains text content that automatically updates multiple times without user intervention and there is no [instrument][] available to stop, pause, hide or alter the frequency of the automatic updates.
+
+```html
+<body onload="startUpdates()">
+	<p>
+		The W3C Web Accessibility Initiative (WAI) develops standards and support materials to help you understand and
+		implement accessibility.
+	</p>
+
+	<p>Random number: <span id="target">1</span></p>
+
+	<script type="text/javascript" src="/test-assets/efbfc7/script.js"></script>
+</body>
+```
+
+#### Failed Example 2
+
+This `span` element contains text content that automatically updates multiple times without user intervention and there is a button available to stop the automatic updates, but the button only stops the updates while it is focused.
+
+```html
+<body onload="startUpdates()">
+	<p>
+		The W3C Web Accessibility Initiative (WAI) develops standards and support materials to help you understand and
+		implement accessibility.
+	</p>
+
+	<p>Random number: <span id="target">1</span></p>
+	<input type="button" onfocus="stopUpdates()" onblur="startUpdates()" value="Stop updates" />
+
+	<script type="text/javascript" src="/test-assets/efbfc7/script.js"></script>
+</body>
+```
+
+### Inapplicable
```

Comment:
@Jym77 There are several examples of elements that do not meet the **no child change** condition in the passed and failed examples. Those examples' descriptions point that out. I'm not really sure how I could come up with an example that would be fully inapplicable (because that child that changes would be applicable).

Good catch for the no text content example missing. I'm going to add it.

Also, I'll update descriptions and order.